game/atl_text_tag.rpy

```diff
@@ -173,11 +173,12 @@ init python:
             def none_to_float(param):
                 if param is None:
                     return 0.0
-                return param
+                return float(str(position(param).relative))
```

Comment:
From my understanding, calling `position()` with only one argument [can lead to unspecified behavior](https://www.renpy.org/doc/html/style_properties.html#position). Also, render.blit [takes an amount of pixels](https://www.renpy.org/doc/html/cdd.html#renpy.Render.blit), so I believe we want absolute position, not relative. I propose:
```
def none_to_float(param):
    if param is None:
        return 0.0
    try:
        return param + 0.0 # ensure float compatibility
    # Since https://www.renpy.org/release/8.2.0, param may be a position object
    except TypeError:
        return param.absolute
```